ta/ta.mk

```diff
@@ -39,9 +39,11 @@ base-prefix :=
 incdirs-host := $(filter-out lib/libutils%, $(incdirs$(sm)))
 incfiles-extra-host := lib/libutils/ext/include/compiler.h
 incfiles-extra-host += lib/libutils/ext/include/util.h
+incfiles-extra-host += lib/libutils/ext/include/types_ext.h
```

Comment:
Wouldn't it be better if `signed_hdr.h` would include `<inttypes.h>` instead of `<types_ext.h>`? It doesn't need more it seems, and you could drop the above line.